vcsh.in

```diff
@@ -320,6 +320,9 @@ list() {
 list_has_remote() {
 	for VCSH_REPO_NAME in $(list); do
 		GIT_DIR=$VCSH_REPO_D/$VCSH_REPO_NAME.git; export GIT_DIR
+		# We can't rely of `git config` both checking for remote and remote
+		# branches requires you to know the name beforehand. This approach
+		# smells weird, but it should be robust.
 		[ -d "$GIT_DIR/refs/remotes/"* ] && echo "$VCSH_REPO_NAME"
```

Comment:
What about `[ -n "$(git remote)" ]`? Not saying it's better, just asking...

Reply:
Never mind my previous suggestion, lets go with the `git for-each-ref ...` option instead and use the return code instead of processing the output.